.github/workflows/ci.yml

```
@@ -0,0 +1,227 @@
+name: CI
+
+on:
+  push:
+    branches: [ main, develop ]
+  pull_request:
+    branches: [ main, develop ]
+
+env:
```

actionlint (1.7.7)

expecting a single ${{...}} expression or mapping value for "env" section, but found plain text node

(syntax-check)